tests/reducers/test.app-reducer.js

```diff
@@ -52,4 +52,28 @@ describe('App Reducer', function() {
     }));
   });
 
+  it('should add a notification', function() {
+    var app = appReducer(initialAppState, {
+      type: actionTypes.ADD_NOTIFICATION,
+      data: {text: 'foo', type: 'info'},
+    });
+    assert.equal(app.notifications.length, 1);
+    assert.equal(app.notifications[0].length, 2);
+    assert.deepEqual(app.notifications[0][1],
+                     {text: 'foo', type: 'info'});
+  });
+
+  it('should remove a notification', function() {
+    var app = appReducer(initialAppState, {
+      type: actionTypes.ADD_NOTIFICATION,
+      data: {text: 'foo', type: 'info'},
+    });
+    var id = app.notifications[0][0];
+    var app = appReducer(initialAppState, {
+      type: actionTypes.REMOVE_NOTIFICATION,
+      id: id,
+    });
+    assert.equal(app.notifications.length, 0);
+  });
```

Comment:
it would be nice to add a test that tries to remove a non-existant notice just to make sure there's no exception